common/script_values/wc_health_values.txt

```diff
@@ -0,0 +1,52 @@
+﻿female_reproduction_age_value = {
+	value = define:NChildbirth|MAX_FEMALE_REPRODUCTION_AGE
```

Comment:
I think due to `MAX_FEMALE_REPRODUCTION_AGE` not changed too old females are still infertile.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/113907658-b03d1f00-97e6-11eb-80b4-9aa7d94e6343.png)

</details>